xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/jackson/modules/Xrpl4jModule.java

```diff
@@ -38,5 +39,7 @@ public Xrpl4jModule() {
 
     addSerializer(LedgerIndex.class, new LedgerIndexSerializer());
     addDeserializer(LedgerIndex.class, new LedgerIndexDeserializer());
+
+    addDeserializer(Transaction.class, new TransactionDeserializer());
```

Comment:
I think we're missing `TransactionResultDeserializer`?

Reply:
I have it in an annotation on `TransactionResult`. Tried adding it here, but it wasn't getting registered properly, then I checked the javadoc for `addDeserializer`
```
/**
     * Method for adding deserializer to handle specified type.
     *<p>
     * WARNING! Type matching only uses type-erased {@code Class} and should NOT
     * be used when registering serializers for generic types like
     * {@link java.util.Collection} and {@link java.util.Map}.
     */
```

Not sure if there is a good way to add a deserializer with a generic type, but using the annotation seems to work fine, so I think I'll just leave it